packages/components/src/Checkbox/Checkbox.mdx

```diff
@@ -27,6 +27,7 @@ import { Checkbox } from "@jobber/components/Checkbox";
         checked={checked}
         label="Save card for future use"
         onChange={setChecked}
+        description="If you save your card now, you'll never need to save it again."
```

Comment:
This doesn't have to block this PR, but could we pull the description from the default example and highlight it later on a la **Indeterminate**? I think the first example of a component that we show should focus on the most likely implementation.